libarchive/test/test_write_format_gnutar.c

```diff
@@ -93,6 +93,29 @@ static const char *longhardlinkname = "Yabcdefghijklmnopqrstuvwxyz"
     "12345678901234567890123456789012345678901234567890"
     "12345678901234567890123456789012345678901234567890";
 
+static const char *longfilename_largeuid = "large_uid_gid---"
```

Comment:
Change the large UID/GID test to _also_ have a long filename.  This tries to reproduce the kind of problem that was reported in #2434.  (Libarchive's gnutar writer does not seem to suffer from this problem, fortunately.)